backend/development.ini.sample

```diff
@@ -289,7 +289,7 @@ email.notification.created_account.subject = [{website_title}] Someone created a
 email.notification.share_content_to_emitter.subject = [{website_title}] you shared {content_filename} to {nb_receivers} users
 email.notification.share_content_to_receiver.subject = [{website_title}] {emitter_name} shared content {content_filename} with you
 email.notification.upload_permission_to_emitter.subject = [{website_title}] you give upload permission on {workspace_name} to {nb_receivers} users
-email.notification.upload_permission_to_receiver.subject = [{website_title}] {emitter_name} give you access to upload content into {workspace_name} share space
+email.notification.upload_permission_to_receiver.subject = [{website_title}] {emitter_name} give you access to upload content into share space {workspace_number}
```

Comment:
replace `give you access to upload content into share space` by `allowed you to upload files in this shared space "{workspace_number}"`

Reply:
done, french translation ?

Reply:
`vous a autorisé à téléverser des fichiers dans l'espace partagé "{workspace_number}"`